app/components/partner/export-stats/index.js

```diff
@@ -55,7 +54,7 @@ export default class PartnerExportStatsComponent extends Component {
       element.click();
       document.body.removeChild(element);
     } catch (err) {
-      this.notify.error(parseError(err));
+      return;
```

Comment:
Since this try is not doing anything lets remove and let the framework handle it if it fails.

Reply:
Alright. I'll tend to this.